exporter/diagnostic_data_collector_test.go

```diff
@@ -77,45 +77,48 @@ mongodb_oplog_stats_wt_transaction_update_conflicts 0` + "\n")
 }
 
 func TestAllDiagnosticDataCollectorMetrics(t *testing.T) {
+	if inGithubActions() {
+		t.Skip("Test not reliable in Gihub Actions")
```

Comment:
Maybe it's not a good idea to rewrite tests if they are not reliable in Github actions.

Reply:
Can we make some queries in mongodb server and try to get full list of metrics?

Reply:
I already tried and it is different in github actions and locally

Reply:
Updated. Created a base list of metrics, common to all versions.